The Long Dark Save Editor 2/Tabs/MapTab.xaml.cs

```diff
@@ -19,6 +19,7 @@ public partial class MapTab : UserControl
         private Point playerPosition;
 
         private string region;
+        private bool doNotRedrawMap;
```

Comment:
I don't think this component actually needs to modify `Boot.m_SceneName` so if we remove it we don't need this.